docs/documentation/full-text/fuzzy.mdx

```diff
@@ -7,6 +7,11 @@ canonical: https://docs.paradedb.com/documentation/full-text/fuzzy
 Fuzziness allows for tokens to be considered a match even if they are not identical, allowing for typos
 in the query string.
 
+<Warning>
+While fuzzy matching will work for non-latin characters (Chinese, Japanese, Korean, etc..), it may not 
+give expected results (with large result sets returned) as Levenshtein distance relies on individual character difference. 
+</Warning>
```

Comment:
So, IMO: we should link to a ticket here about expanding our support for fuzzy search in those languages, or ask that they file an issue if they want it.

There are libraries for doing edit distance with each of these languages, but also, it's possible that rather than fuzzy matching, what users really want out of this is language-model based spell correction.

cc @mdashti : This might be an interesting part of improving our fuzzy matching.